### PR TITLE
Heroku setup

### PR DESCRIPTION
**Pull request by kerzhner**
These are the basic setup steps for Heroku. I can't test these changes with the latest `master` as there are Typescript errors.

At the moment deploys are manual using Heroku CLI. We should decide whether want a git branch to automatically deploy to Heroku.

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ const env = require('dotenv').config();
 let dotenvExpand = require('dotenv-expand');
 dotenvExpand(env);
 
-const BASE_PATH = path.join(__dirname, 'src', 'wallet', 'db');
+const sourceDirectory = 'src';
+if (process.env.NODE_ENV === 'production') {
```
Comment (andrewgordstewart) [inline knexfile.js:9]: This is a bit pedantic, but I think we should call the current heroku environment staging for now?

File: package.json [hunk 2]
```diff
@@ -11,17 +11,23 @@
...
   "dependencies": {
+    "dotenv": "^6.2.0",
+    "dotenv-expand": "^4.2.0",
     "fmg-core": "^0.5.6",
     "fmg-nitro-adjudicator": "^0.0.10",
     "knex": "^0.13.0",
     "koa": "^2.7.0",
     "koa-body": "^4.0.8",
     "koa-router": "^7.4.0",
+    "magmo-devtools": "git+https://github.com/magmo/devtools.git#v0.1.8",
```
Comment (andrewgordstewart) [inline package.json:30]: This doesn't seem like it should be a dependency, but let's go with it for now.

**Review by andrewgordstewart — APPROVED** [review]

**Comment by andrewgordstewart** [comment]
(I force-pushed after rebasing onto master)

File: knexfile.js (hunk 1 above)
Comment (kerzhner) [reply knexfile.js:9]: @andrewgordstewart  Agreed that we need a staging environment. Currently, our Heroku app is a named `rps-autoplayer`. If we want to use this app as the staging environment, we should probably rename it to `rps-autoplayer-staging`. The other alternative, is to use the the current app for production and spin up a new staging app.

File: package.json (hunk 2 above)
Comment (kerzhner) [reply package.json:30]: @andrewgordstewart we are using the dependency here: https://github.com/magmo/node-bot/blob/master/src/wallet/utilities/blockchain.ts#L6. That's why it was moved from `devDependencies` to `dependencies`.

Comment (andrewgordstewart) [reply package.json:30]: Yes, but it's a needless dependency that should probably be removed.

**Pull request merged**